File: testresources/csharp/benchmark/Domain/Direct/Violating/CallInstanceLibraryClass.cs
using Domain.Direct;


namespace Domain.Direct.Violating
{
	using FI.Foyt.Foursquare.Api;

	public class CallInstanceLibraryClass : Base
	{
		protected internal FoursquareApi fourApi;

		public void HandleCallback()
		{
			fourApi.GetAuthenticationUrl();
		}
	}
}

File: testresources/java/accuracy/src/domain/direct/violating/CallInstanceLibraryClass.java
package domain.direct.violating;

import fi.foyt.foursquare.api.FoursquareApi;
import fi.foyt.foursquare.api.FoursquareApiException;

public class CallInstanceLibraryClass {
	private FoursquareApi fourApi;
	
	public void handleCallback() {
				fourApi.getAuthenticationUrl();
	}
}